### DISCUSSION TITLE
How to use 2 git users on mac?

### DISCUSSION THREAD

**Discussion by ntmduy** [community/community#15244] [Projects and Issues]
I everyone, I have a question need you help.

I have 1st user account using for companyA
and I have 2nd user account using for companyB

So I only pull & push 1st user account to repo of companyA, and 2nd user account to repo of companyB, in 1 machines Macbook.
Please help me how to setup this, or show me exist giude entry on community.

Thank you.

**Comment by Zi-Tao** [comment]
@ntmduy https://gist.github.com/Jonalogy/54091c98946cfe4f8cdab2bea79430f9

**Comment by huabin** [comment]
Hello @ntmduy,

If you are trying to do multi-user operations in the terminal, there's only one thing you need to do.

- Add the username to each repo's remote link 

E.g. usernameA for companyA

```
git remote add origin https://github.com/companyA/reponame.git
```

Should be 

```
git remote add origin https://usernameA@github.com/companyA/reponame.git
```

UsernameB does the same.

```
git remote add origin https://github.com/companyB/reponame.git
```

Should be 

```
git remote add origin https://usernameB@github.com/companyB/reponame.git
```

Hope this can help you.

**Comment by linhmy317** [comment]
Hello Duy,
Please check my post.
